TF_Tooling_Web/src/TouchFree.ts

```diff
@@ -0,0 +1,24 @@
+import { ConnectionManager } from "./Connection/ConnectionManager";
+import { SVGCursor, SVGCursorParams } from "./Cursors/SvgCursor";
+import { TouchlessCursor } from "./Cursors/TouchlessCursor";
+import { WebInputController } from "./InputControllers/WebInputController";
+
+var InputController: WebInputController;
+export var CurrentCursor : TouchlessCursor;
+
+export function Init(_cursor: TouchlessCursor): void;
+export function Init(_svgParams?: SVGCursorParams): void;
+export function Init(_cursorOrParams?: SVGCursorParams | TouchlessCursor): void {
+    ConnectionManager.init();
+
+    InputController = new WebInputController();
+
+    if (_cursorOrParams instanceof TouchlessCursor) {
+        CurrentCursor = _cursorOrParams;
+    }
+    else if (_cursorOrParams !== undefined) {
+        CurrentCursor = SVGCursor.FromParamObj(_cursorOrParams);
+    } else {
+        CurrentCursor = new SVGCursor();
+    }
+}
```

Comment:
I would suggest having the signature be an optional interface with optional properties so we can extend it in future

```suggestion
type CursorInit = TouchlessCursor | SVGCursorParams;

interface InitParams {
    Cursor?: CursorInit
}

export function Init(params?:InitParams): void {
    ConnectionManager.init();

    InputController = new WebInputController();

    if (params?.Cursor === undefined)
    {
        CurrentCursor = new SVGCursor();
    }
    else if (params.Cursor instanceof TouchlessCursor) {
        CurrentCursor = params.Cursor;
    }
    else
    {
        CurrentCursor = SVGCursor.FromParamObj(params.Cursor);
    }
}
```

Reply:
Also if we end up with a lot of init options this is cleaner and easier to understand by keeping the actual Init function small (rather than loads of parameters)

Reply:
This is done now, though we now only have one option (cursor / no)

Reply:
I think what we have here is a good balance between "an easy way to initialise touchfree" and "I want control of a cursor", wherein if you want control, you get full control.

Honestly the "CurrentCursor" reference here is the bit I have issue with; noone benefits from the Tooling having control of a cursor instance, it's only there in the case that you want to change the cursor you got defaulted. Ideally in most cases people will use the version that doesn't init a cursor and then init their own, and customise the cursors seperately.

Reply:
> we lose the flexibility of initializing with custom values

We aren't removing this flexibility, we're just pushing responsibility for initialising the custom parts that don't benefit from being controlled by tooling onto users. This gives us two scenarios:

1. You want it easy & fast, you get the default cursor
2. You want a custom cursor, so you make your own cursor (which is still _very_ simple)